data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/Main.java

```diff
@@ -155,6 +155,7 @@ public static void main(final String[] args) throws IOException {
       logger.error("Failed during filesystem watch", ex);
 
       Shutdown.closeSync(vertx).run();
+      sdkTracerProvider.close();
```

Comment:
I think we should close this even if there are no exception.

Reply:
Some line above we add a shutdown hook.

Reply:
Same question for the receiver.

Reply:
I did a little fancy change to Shutdown hook, check it out